packages/nimble-components/src/select/styles.ts

```diff
@@ -151,6 +151,28 @@ export const styles = css`
         align-items: center;
         padding: ${smallPadding} ${mediumPadding};
     }
+
+    .loading-container {
+        padding-left: ${mediumPadding};
+        padding-right: ${mediumPadding};
+        display: flex;
+        height: ${controlHeight};
+        align-items: center;
+    }
+
+    .loading-container.above {
+        margin-top: ${mediumPadding};
+        margin-bottom: 0;
+    }
+
+    .loading-container.below {
+        margin-top: 0;
+        margin-bottom: ${mediumPadding};
+    }
```

Comment:
I don't see the loading container ever getting a class of `above` or `below`.

Reply:
Yep. This, and the fact that the styles were wrong as-is, was the reason for the vertical padding issues you mentioned.